client/manager_sql.go

```diff
@@ -213,7 +231,8 @@ func (d *sqlData) ToClient() (*Client, error) {
 		FrontChannelLogoutSessionRequired: d.FrontChannelLogoutSessionRequired,
 		PostLogoutRedirectURIs:            stringsx.Splitx(d.PostLogoutRedirectURIs, "|"),
 		BackChannelLogoutURI:              d.BackChannelLogoutURI,
-		BackChannelLogoutSessionRequired:  d.BackChannelLogoutSessionRequired,
+		BackChannelLogoutSessionRequired:  d.BackChannelLogoutSessionRequired, //
+		Metadata:                          metadata,
```

Comment:
If `metadata` is `json.RawMessage` there would be no need for decoding as we could do:

```suggestion
		Metadata:                          []byte(metadata),
```

Then, all previous comments (Coalesce etc) would no longer be required.

Reply:
I made this change